articles/quickstart-microsoft-provider-format.md

```diff
@@ -284,83 +284,80 @@ try:
     import pulser
 except ImportError:
     !pip -q install pulser
-    !pip -q install pulser-core
 ```
 
 #### Create a quantum register
 
-1. First, load the required imports:
+1. First, you create a 'devices' object to import the PASQAL quantum computer target, Fresnel. FRESNEL provides predefined trap layouts that you can utilize to create your quantum register. To configure your quantum registers, you arrange an array of qubits.
```

Comment:
Is it Fresnel, FRESNEL, or Fresnel1? Our docs show [Fresnel1](https://learn.microsoft.com/en-us/azure/quantum/provider-pasqal#fresnel1)